Source/Diagnostics/ReducedDiags/Costs.H

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2019-2020 Yinjian Zhao
```

Comment:
```suggestion
/* Copyright 2019-2020 Michael Rowan
```

Reply:
I add both our names, since either way we now both contribute directly to these files :-)